app/components/views/GetStartedPage/Page.js

```diff
@@ -3,14 +3,20 @@ import DecredLoading from "../../DecredLoading";
 import "../../../style/GetStarted.less";
 import "../../../style/Layout.less";
 
-const Page = ({ Header, Body, ...props }) => (
-  <div className="page-view inverted-colors">
-    <Header {...props} />
-    <div className="page-content-fixed">
-      <DecredLoading hidden={!props.isProcessing} className="get-started-loading" />
-      <Body {...props} />
+const Page = ({ Header, Body, ...props }) => {
+  const { isAdvancedDaemon, isSubmited, hasErrors} = props;
+  return (
+    <div className="page-view inverted-colors">
+      <Header {...props} />
+      <div className="page-content-fixed">
+        {
+          isAdvancedDaemon ?
+            isSubmited && !hasErrors ? <DecredLoading hidden={!props.isProcessing} className="get-started-loading" /> : null
+            : <DecredLoading hidden={!props.isProcessing} className="get-started-loading" />
+        }
+        <Body {...props} />
+      </div>
```

Comment:
You shouldn't be making any changes here.  If you need it to take those props into account add a new selector and then add that to the isStartupProcessing selector (refer to isPrepared) for an example of this.  But this Page.js should be unchanged for your needs.